test/mailers/mailer_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
```

Comment:
Is there any reason to keep this in separate file? There are already tests in `test/unit/mailer_test.rb`. I'm happy to move those to `test/mailers` if that's the default path for generating mailer tests in Rails today (I can do this in separate PR on my own).